ocm/__init__.py

```diff
@@ -211,10 +211,19 @@ class ResourceRelation(enum.StrEnum):
     EXTERNAL = 'external'
 
 
+@dc
+class LabelMerge:
```

Comment:
nit: would it be more expressive to name this `LabelMergeSpec` (or just MergeSpec) - would also match the term used in [ocm-spec](https://github.com/open-component-model/ocm-spec/blob/main/doc/01-model/03-elements-sub.md#labels)